libvirt/resource_libvirt_domain.go

```diff
@@ -531,7 +531,16 @@ func resourceLibvirtDomainCreate(d *schema.ResourceData, meta interface{}) error
 	if len(waitForLeases) > 0 {
 		err = domainWaitForLeases(domain, waitForLeases, d.Timeout(schema.TimeoutCreate), d)
 		if err != nil {
-			return err
+
+			ipNotFoundMsg := "Error couldn't retrieve IP address of domain." +
+				"Please check out as possible tips: " +
+				"1) is the domain running proplerly? " +
+				"2) has the network interface an IP address? " +
+				"3) Networking issues on you KVM setup? " +
```

Comment:
I would skip KVM and suggest libvirt:

```suggestion
				"3) Networking issues on your libvirt setup? " +
```